docs/howtoguides/enable_in_dockerfile.md

```diff
@@ -49,41 +50,41 @@ FROM ubuntu:focal
 # In the next step, we demonstrate how to pass the file as a secret when
 # running docker build.
 RUN --mount=type=secret,id=pro-attach-config \
-
+    #
     # First we update apt so we install the correct versions of packages in
     # the next step
     apt-get update \
-
+    #
     # Here we install `pro` (ubuntu-advantage-tools) as well as ca-certificates,
     # which is required to talk to the Ubuntu Pro authentication server securely.
     && apt-get install --no-install-recommends -y ubuntu-advantage-tools ca-certificates \
-
+    #
     # With pro installed, we attach using our attach config file from the
     # previous step
     && pro attach --attach-config /run/secrets/pro-attach-config \
-
+    #
     ###########################################################################
     # At this point, the container has access to all Ubuntu Pro services
     # specified in the attach config file.
     ###########################################################################
-
+    #
     # Always upgrade all packages to the latest available version with the Ubuntu Pro
     # services enabled.
     && apt-get upgrade -y \
-
+    #
     # Then, you can install any specific packages you need for your docker
     # container.
     # Install them here, while Ubuntu Pro is enabled, so that you get the appropriate
     # versions.
     # Any `apt-get install ...` commands you have in an existing Dockerfile
     # that you may be migrating to use Ubuntu Pro should probably be moved here.
     && apt-get install -y openssl \
-
+    #
     ###########################################################################
-    # Now that we've upgraded and installed any packages from the Ubuntu Pro
+    # Now that we have upgraded and installed any packages from the Ubuntu Pro
```

Comment:
These grammatical changes are to avoid `'`s in the comments, which were causing the syntax highlighting to fail